src/views/Orders.vue

```diff
@@ -0,0 +1,347 @@
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+              </div>
+              <div v-else>
+                <ion-item>
+                  <ion-label> {{ $t("Shipping method") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+                <ion-item>
+                  <ion-label> {{ $t("Shipping from") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+                <ion-item>
+                  <ion-label> {{ $t("Location inventory") }} </ion-label>
+                  <!-- Need to work on it -->
+                  <p slot="end"> Not available </p>
+                </ion-item>
+              </div>
+            </ion-card>
+          </div>
+        </div>
+      </div>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script>
+import {
+  IonBackButton,
+  IonBadge,
+  IonButton,
+  IonCard,
+  IonCardContent,
+  IonCheckbox,
+  IonChip,
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonNote,
+  IonPage,
+  IonSearchbar,
+  IonSelect,
+  IonThumbnail,
+  IonTitle,
+  IonToolbar,
+  IonSelectOption,
+} from "@ionic/vue";
+import { pricetag, ribbon } from "ionicons/icons";
+import { useStore } from "@/store";
+import { mapGetters } from "vuex";
+import { showToast } from '@/utils'
+import { Plugins } from '@capacitor/core';
+import Image from '@/components/Image.vue';
+
+const { Clipboard } = Plugins;
+
+export default {
+  name: "Order",
+  components: {
+    IonBadge,
+    IonBackButton,
+    IonButton,
+    IonCard,
+    IonCardContent,
+    IonChip,
+    IonCheckbox,
+    IonContent,
+    IonHeader,
+    IonItem,
+    IonIcon,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonNote,
+    IonPage,
+    IonSelect,
+    IonSelectOption,
+    IonSearchbar,
+    IonThumbnail,
+    IonTitle,
+    IonToolbar,
+    Image
+  },
+
+  computed: {
+    ...mapGetters({
+      orders: 'order/getList',
+      getProduct: 'product/getProduct',
+      currentFacilityId: 'user/getCurrentFacility'
+    })
+  },
+
+  methods: {
+    async getOrders(vSize, vIndex){
+      const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
+      const viewIndex = vIndex ? vIndex : 0;
+      const payload = {
+        viewIndex: viewIndex,
+        viewSize: viewSize,
+        queryString: '* *',
+        groupLimit: 10000,
+        groupByField: 'orderId',
+        filters: JSON.parse(process.env.VUE_APP_ORDER_FILTERS)
+      }
+      await this.store.dispatch("order/findOrders", payload)
+    },
+
+    async copyToClipboard(text) {
+      await Clipboard.write({
+        string: text
+      }).then(() => {
+        showToast(this.$t('Copied', { text }));
+      })
+    },
+
+  },
+  mounted() {
+    this.getOrders(10, 0);
```

Comment:
We won't pass the view size and view index from the mounted hook

Reply:
Okay, Sir

Reply:
Removed but this was optional for testing purpose